packages/fast-components-react-msft/src/slider/examples.data.tsx

```diff
@@ -65,6 +69,14 @@ const examples: ComponentFactoryExample<SliderProps> = {
     },
     data: [
         {
+            range: {
+                maxValue: 100,
```

Comment:
These are default values so not necessary unless you want them there for clarity.